lib/vsc/administration/vo.py

```diff
@@ -211,7 +209,18 @@ def _create_fileset(self, filesystem_name, path, parent_fileset=None, fileset_na
 
         The parent_fileset is used to support older (< 3.5.x) GPFS setups still present in our system
         """
-        self.gpfs.list_filesets()
+        fs_backend = load_storage_operator(storage)
+
+        try:
+            filesystem_name = storage.filesystem
+        except AttributeError:
+            logging.exception("Trying to access non-existent attribute 'filesystem' in the data storage instance")
```

Comment:
see remarks above. this should be raised i think

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/d9d2f11e38cfec377807ac56339b0498d1b1400d